apps/communications/dialer/js/oncall.js

```diff
@@ -8,6 +7,9 @@
 function onLoadCallScreen(evt) {
   window.removeEventListener('load', onLoadCallScreen);
 
+  // According to the UX sound spec, we should use the "system" type for the
+  // dialer pad. See the attachment of the bug1068219.
+  TonePlayer.init('telephony');
```

Comment:
Where we use this telephony type for? 
To generate the end sound ? or other purposes?